pkg/internal/codegen/parse/util.go

```diff
@@ -395,3 +425,45 @@ func parseScaleParams(t *types.Type) (map[string]string, error) {
 	}
 	return nil, fmt.Errorf(jsonPathError)
 }
+
+// ready subresource requires name,type,JSONPath fields and rest of the field are optional
+// +kubebuilder:printcolumn:ready:name=<name>,type=<type>,description=<desc>,JSONPath:<.spec.Name>,priority=<0|1>,format=<format>
+func parsePrintColumnParams(t *types.Type) v1beta1.CustomResourceColumnDefinition {
+	config := v1beta1.CustomResourceColumnDefinition{}
+	for _, comment := range t.CommentLines {
+		if strings.Contains(comment, "+kubebuilder:printcolumn:ready") {
```

Comment:
`ready` is not required. here and in other places.

Reply:
ack